frontend/kesaseteli/youth/src/components/header/__tests__/Header.test.tsx

```diff
@@ -0,0 +1,21 @@
+import Header from 'kesaseteli/youth/components/header/Header';
+import headerApi from 'kesaseteli-shared/__tests__/utils/component-apis/header-api';
+import renderComponent from 'kesaseteli-shared/__tests__/utils/components/render-component';
+import React from 'react';
+import { waitFor } from 'shared/__tests__/utils/test-utils';
+import { SUPPORTED_LANGUAGES } from 'shared/i18n/i18n';
+
+describe('frontend/kesaseteli/youth/src/components/header/Header.tsx', () => {
+  it('can change supported language', async () => {
+    const spyRouterPush = jest.fn();
+    renderComponent(<Header />, { push: spyRouterPush });
+    for (const lang of SUPPORTED_LANGUAGES) {
```

Comment:
I'd rather use a more functional approach. SUPPORTED_LANGUAGES.forEach(...

Reply:
async is not supported in forEach. I could use `promise.all` but somehow I found this old school way is more readable in this case :)